printer_mods/mpfl/V0.1_skirt_OLED/README.md

```diff
@@ -0,0 +1,10 @@
+# V0.1 0.96" OLED skirt
+
+![Crappy photo](photo.jpg)
+
+Inspired by [`madros/V0_skirt_OLED`](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/madros/V0_skirt_OLED), but for the V0.1. Please see that directory for installation instructions - no need to repeat them here.
+
+OpenSCAD file provided if you want to customise (perhaps you'd like to replace that Voron logo with your serial).
+You will need to grab [`Cowling_LOGO_ONLY_x1.STL`](https://github.com/VoronDesign/Voron-0/blob/V0.1/STLs/Toolheads/Mini_Afterburner/Cowling_LOGO_ONLY_x1.STL) and [`Skirt_x3.STL`](https://github.com/VoronDesign/Voron-0/blob/V0.1/STLs/Skirt_x3.STL) from [`VoronDesign/Voron-0`](https://github.com/VoronDesign/Voron-0/releases/tag/V0.1).
+
+v0.1-skirt-oled.scad is licensed with a GNU General Public License v3.0. All other file used for this mod are copyright their respective creators and are licensed separately.
```

Comment:
Please remove this paragraph, this whole repository is licensed GPL 3, which applies to all mods in here.

https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md

Reply:
Fixed in ffe868a